kernel/aster-nix/src/util/random.rs

```diff
@@ -22,8 +22,22 @@ pub fn init() {
     cfg_if::cfg_if! {
         if #[cfg(target_arch = "x86_64")] {
```

Comment:
Since you have a `ostd::arch::read_random` implementation. This conditional compilation can be removed now.